ykrt/src/compile/jitc_yk/trace_builder.rs

```diff
@@ -0,0 +1,56 @@
+//! The trace builder.
+//!
+//! Given a mapped trace and an AOT module, assembles an in-memory Yk IR trace by copying blocks
+//! from the AOT IR. The output of this process will be the input to the code generator.
+
+use super::aot_ir::{self, Module};
+use crate::trace::{MappedTrace, TracedAOTBlock};
+use std::error::Error;
+
+struct Grafter<'a> {
```

Comment:
I guess this should be `TraceBuilder` now?

Reply:
Huh! My grep-fu fails me!